.github/workflows/release.yml

```
@@ -0,0 +1,46 @@
+---
+name: Release
+
+on:  # yamllint disable-line rule:truthy
+  release:
+    types: [published]
+
+jobs:
+  release:
+    name: Create Release
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+    - name: Set up QEMU
+      id: qemu
+      uses: docker/setup-qemu-action@v2
+      with:
+        platforms: linux/amd64,linux/arm64
+    - name: Setup buildx
+      uses: docker/setup-buildx-action@v2
+    - name: Get tag
+      id: tag
+      uses: dawidd6/action-get-tag@v1
+      with:
+        strip_v: false
+    - name: Docker meta
+      id: meta
+      uses: crazy-max/ghaction-docker-meta@v4
```

This seems to already redirect to `https://github.com/docker/metadata-action` so maybe we use that?

```suggestion
      uses: docker/metadata-action@v4
```

---

++